modules/web-console/src/main/java/org/apache/ignite/test/WebConsoleLauncher.java

```diff
@@ -56,7 +59,21 @@ public static void main(String... args) {
         RestApiRouter cfgsRouter = new ConfigurationsRouter(ignite);
         RestApiRouter notebooksRouter = new NotebooksRouter(ignite);
 
-        vertx.deployVerticle(new WebConsoleServer(ignite, auth, cfgsRouter, notebooksRouter));
+        WebConsoleConfiguration cfg = new WebConsoleConfiguration();
+
+        // Uncomment if you need Vertx to handle static resources.
```

Comment:
Why we comment this code?

Reply:
Because it is just for development for now, will be fixed by WC-950, added TODO in code